src/static/css/almanac.css

```diff
@@ -387,6 +386,38 @@ h2.header {
   font-size: 0.875rem;
   text-decoration: none;
   text-align: center;
+  box-shadow: 2px 2px 3px 3px lightgray;
```

Comment:
Can we add this box-shadow just to the `chapter-cta` class rather than all btns? Looks a bit weird for the buttons on the home screen.